src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -99,6 +99,7 @@
                     :selections="comparisonPlotSelections"
                     :formatFunction="formatBarchartValue"
                     :showRangesInTooltips="true"
+                    :no-data-message="noChartData"
```

Comment:
I tried using `v-translate:no-data-message="'noChartData'"` but this wouldn't work properly as a prop. I suspect it had something to do with the order of operations.